idriss-xyz/core
# fix: Add an RPC for public ethereum client (fix cloudflare ENS res)

## Overview
There was an error that ocurred when tryong to donate to a creator with their ENS. The resolution step was failing due to cloudflare-eth servers poorly mantained. Error shown in browser console when navigating to [donate page with ENS](https://www.idriss.xyz/creators/donate?address=qflux.eth&token=ETH,USDC,DAI,PRIME,GHST,YGG,PDT,DEGEN&network=Aleph,Base,Ethereum,Mantle,Optimism,Polygon&creatorName=qFlux888) : `ContractFunctionExecutionError: Requested resource not found.  URL: https://cloudflare-eth.com/`

## How it was fixed
I added an Ethereum mainet RPC URL in an `.env` file (added `.env.example` as a guide) in the main `main-landing` folder. This is a personal temporary API key that should be replaced soon (I will edit this description if it was added).

> EDIT
I added a public RPC instead (https://eth.llamarpc.com) of a private one. Reasons: People can always check the networks tab, and should be enough for IDRISS.

## How to test
Go to https://www.idriss.xyz/creators/donate?address=qflux.eth&token=ETH,USDC,DAI,PRIME,GHST,YGG,PDT,DEGEN&network=Aleph,Base,Ethereum,Mantle,Optimism,Polygon&creatorName=qFlux888 and see no errors pop up in the console
